end-to-end-computer-vision/README.md

```diff
@@ -32,7 +32,8 @@ deployed instance of zenml:
 
 ```bash
 pip install -r requirements.txt
-zenml integration install label_studio torch gcp mlflow -y
+zenml integration install torch gcp mlflow -y
+pip install label-studio
```

Comment:
```suggestion
```

I think this is no longer needed? And maybe you can add label_studio integration here too instead of below?